### DISCUSSION TITLE
如何进行渲染

### DISCUSSION THREAD

**Discussion by yjwan1314** [nilsschmidt1337/ldparteditor#1112] [General]
我尝试用这个软件进行渲染，但是我不知道如何去选中这些面。希望能有大佬来指导一下
![Uploading 898fa5e84511b5a32bb302708a9651a.png…]()

**Comment by nilsschmidt1337** [comment]
@yjwan1314, 请看这里寻求帮助：
https://1-jc--tchang-philohome-com.translate.goog/manuel/LDPartEditor.htm?_x_tr_enc=1&_x_tr_sl=auto&_x_tr_tl=zh-CN&_x_tr_hl=de&_x_tr_pto=wapp
